jdaviz/configs/specviz/helper.py

```diff
@@ -80,7 +80,18 @@ def get_spectral_regions(self):
         spec_regs = {}
 
         for name, reg in regions.items():
-            unit = reg.meta.get("spectral_axis_unit", u.Unit("Angstrom"))
+            try:
+                # Use the region's unit. If N/A, use the reference_data's
+                unit = reg.meta.get("spectral_axis_unit",
+                                    self.get_spectra(
+                                        self.app.get_viewer("spectrum-viewer"
+                                                            ).state.reference_data.label,
+                                        apply_slider_redshift=False
+                                        ).spectral_axis.unit
+                                    )
+            except (KeyError, LookupError, AttributeError):
+                unit = u.Unit('count')
```

Comment:
Counts usually means uncalibrated flux (of some sort). If this is spectral X-axis, maybe pixel is more appropriate... or simple unitless if pixel is misleading.

Reply:
Yea @camipacifici corrected me in slack. I think I must have hit my head when I put in that except statement. Cami is recruiting @eteq @PatrickOgle @orifox for help deciding on what to fallback to

Reply:
I would also vote for pixels, but I like the unitless idea if it does not break other things along the way.